README.md

```diff
@@ -297,6 +297,33 @@ make load-test
 - Review logs that will be sent to Log Analytics when configured
   - See `deploy/loganalytics` for directions
 
+## How Codespaces is built?
+
+A development container is a fully-featured development environment running in a container. The development container is a Docker container running locally or remotely that, at a high-level, contains the following:
+
+- All the application dependencies - Defined in a Docker image, Dockerfile or docker-compose file and potentially updated via scripts called by well-defined hooks.
+- The application code - mounted, copied or cloned into the container
+- Visual Studio Code Server - configured with the appropriate Visual Studio Code Extensions required to develop
+  
+Codespaces extends the use of development containers by providing a remote hosting environment for them. Developers can simply click on a button in GitHub to open a Codespace for the repo. Behind the scenes, GitHub Codespaces is:
+
+- Spinning up a VM
```

Comment:
I believe it's a container, not a VM

Reply:
@bartr From the [docs](https://docs.github.com/en/codespaces/getting-started/deep-dive#creating-your-codespace), it seems it's spinning up a container inside a new VM for the codespaces. Does this step still makes sense?

Reply:
makes sense